R/convert_ov_wenr.R

```diff
@@ -0,0 +1,37 @@
+#' Convert possible B_OV_WENR values to standardised values
+#' 
+#' This function converts numeric values for OV_WENR to values used by other OBIC functions if numeric values are entered.
+#' 
+#' @param B_OV_WENR () Data on soil compaction risk that may have to be converted to string
+#' 
+#' @import data.table
+#' 
+#' @export
+
+convert_WENR <- function(B_OV_WENR) {
+  # Create data table
+  dt <- data.table(B_OV_WENR = B_OV_WENR)
+  if(is.numeric(B_OV_WENR)){
```

Comment:
When `B_OV_WENR` is a vector this would not work, better to convert it to `ifelse()`?

Reply:
I think if(is.numeric(dt$B_OV_WENR) should work, instead of ifelse() as we expect either numbers of strings.

Reply:
If only looks to the first value in the vector, while the user could make use of both formats for different fields, when the B_OV_WENR is derived from different sources.
Also this will generate a warning, when the `B_OV_WENR` is a vector, which is confusing for the user ;)

Reply:
Removed if entirely in 29ef828 the function should now be able to deal with numeric, string, and mixed data for B_OV_WENR.